hapi-proto/src/main/java/com/hedera/services/legacy/proto/utils/KeyExpansion.java

```diff
@@ -640,7 +640,9 @@ public static void expandKeyMinimum4Signing(Key key, int depth, List<Key> expand
         int i = 0;
         for (Key aKey : tKeys) {
           if(i++ >= thd) // if threshold is reached, stop expanding keys
-            break;
+		  {
+			  break;
```

Comment:
Should we throw some exception to alert this situation. Or at least log a warning message?

Reply:
Makes sense. Adding a debug level log here.